sdk/keyvault/azure-security-keyvault-common/vcpkg/Config.cmake.in

```diff
@@ -4,7 +4,7 @@
 @PACKAGE_INIT@
 
 include(CMakeFindDependencyMacro)
-find_dependency(azure-core-cpp)
+find_dependency(azure-core-cpp "1.1.0")
```

Comment:
```suggestion
find_dependency(azure-core-cpp "1.2.0")
```

Reply:
This is optional as keyvault-common is abandoned

Reply:
I'd rather keep it updated a bit, until we delete it.
Should we delete it from the source tree?